include/ibamr/IBFEMethod.h

```diff
@@ -600,23 +593,19 @@ class IBFEMethod : public IBStrategy
     /*!
      * Register a load balancer and work load patch data index with the IB
      * strategy object.
+     *
+     * This function should be called before IBFEMethod::initializeFEData().
      */
     void registerLoadBalancer(SAMRAI::tbox::Pointer<SAMRAI::mesh::LoadBalancer<NDIM> > load_balancer,
                               int workload_data_idx) override;
 
     /*!
-     * Update work load estimates on each level of the patch hierarchy. If a
-     * load balancer and workload variable have been previously specified by
-     * IBFEMethod::register_load_balancer then the workload_data_idx variable
-     * index must be the same.
-     *
-     * @note This function computes workloads by setting the estimated work
-     * value on all cells to 1 and then using
-     * FEDataManager::updateWorkloadEstimates to compute the estimated
-     * Lagrangian contribution to the total amount of work.
+     * Add the estimated computational work from the current object (i.e., the
+     * work required by the owned Lagrangian objects) per cell into the
+     * specified <code>workload_data_idx</code>.
      */
-    void updateWorkloadEstimates(SAMRAI::tbox::Pointer<SAMRAI::hier::PatchHierarchy<NDIM> > hierarchy,
-                                 int workload_data_idx) override;
+    void addWorkloadEstimate(SAMRAI::tbox::Pointer<SAMRAI::hier::PatchHierarchy<NDIM> > hierarchy,
+                             const int workload_data_idx) override;
```

Comment:
Try to avoid `const int` (or similar) in the header.

Reply:
I am curious; why? I don't know what difference the const really makes.